hack/helm_vars/wire-server/values.yaml.gotmpl

```diff
@@ -78,7 +78,9 @@ brig:
       # See helmfile for the real value
       setFederationDomain: integration.example.com
       setFederationDomainConfigs:
-        # See helmfile for the real value
+        # 'setFederationDomainConfigs' is deprecated as of https://github.com/wireapp/wire-server/pull/3260.  See
```

Comment:
I think it makes sense to set values for CI here, too, for the following settings (even if it's only the default settings):

- `setFederationStrategy`
- `setFederationDomainConfigsUpdateFreq`